ord_schema/proto/reaction.proto

```diff
@@ -916,6 +1025,10 @@ message ReactionAnalysis {
     ED = 23;  // Electron diffraction, including MicroED.
   }
   AnalysisType type = 1;
+  // Whether this analysis is intended to be of an isolated species. If false
+  // or unspecified, the assumption is that it is of a crude product mixture
+  // (or a partially worked-up product mixture).
+  optional bool is_of_isolated_species = 8;
```

Comment:
This is a bit awkward. Possible alternatives:

- `is_isolated_species`
- `isolated_species`

We've (mostly) taken an `is/has/uses/used<X>` naming style for boolean values in the schema so far. I'm wondering if we should take this opportunity to drop these prefixes to make things flow a bit better. IMO a checkbox in the editor that says "desired_product" is at least as (or more) readable than "is_desired_product".

See https://medium.com/javascript-in-plain-english/naming-boolean-variables-prefixes-df973d7d7ec3

Reply:
I like the current naming style tbh. I think `analysis.is_of_isolated_species = True` is clearer than `analysis.isolated_species = True`. Obviously the form can use whatever label we want